lib/data_migrate/database_tasks.rb

```diff
@@ -2,11 +2,12 @@
 
 module DataMigrate
   ##
-  # This class extends DatabaseTasks to add a data_schema_file method.
+  # This class extends DatabaseTasks to add a schema_file method.
   class DatabaseTasks
     extend ActiveRecord::Tasks::DatabaseTasks
 
-    def self.data_schema_file
+    # This overrides ActiveRecord::Tasks::DatabaseTasks
+    def self.schema_file(_format = nil)
```

Comment:
Curious why you changed this to `schema_file `since `data_schema_file` is more descriptive of the file

Reply:
The code was relying on the fact that it was overriding the method from `ActiveRecord::Tasks::DatabaseTasks`. Without this `schema_file` was not being called when `load_schema_current` is called from `tasks/databases.rake db:schema:load`

It is more descriptive but broke the original intent.